src/views/partials/head.hbs

```diff
@@ -1,6 +1,7 @@
 <head>
   <meta charset="utf-8">
-  <title>Elegant</title>
-  <link rel="stylesheet" href="css/login.css">
+  <title>{{title}}</title>
```

Comment:
nice nice ^_^ 💃

Reply:
I like that i will use it :+1:  good job honey

Reply:
:dancer: